ocs_ci/ocs/platform_nodes.py

```diff
@@ -220,6 +223,31 @@ def get_vms(self, nodes):
             vms.extend(node_vms)
         return vms
 
+    def get_vms_in_dc(self, nodes, dc=None):
```

Comment:
The function name is similar to ocs_ci.utility.vsphere.VSPHERE.get_all_vms_in_dc.
I think its name should be OCP nodes oriented, maybe `get_vm_nodes_in_dc`